src/main/java/seedu/multitasky/storage/StorageDateConverter.java

```diff
@@ -0,0 +1,60 @@
+package seedu.multitasky.storage;
+
+import java.text.DateFormat;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Calendar;
+import java.util.Date;
+
+//@@author A0132788U
+/**
+ * Class that implements methods to convert Date to String to enable Storage to store dates in a human-editable form in
+ * the XML file.
+ * Parses String back to Date and returns to Model.
+ */
+public class StorageDateConverter {
+
+    /** Formatter to parse date into a human-editable string to store in the XML file */
+    private DateFormat formatter = new SimpleDateFormat("dd/MM/yy HH:mm");
```

Comment:
but it does show in MM dd yy on the ui, although the month is in word form. 

also the way SimpleDateFormatter works, we should probably do a formatter.setLenient(false) before we use the formatter to parse , because otherwise it will accept strange things like 13 as a month and parse it into 1 month 1 year. so currently if a user inputs MM/dd/yy like 1/13/17 but we are parsing it as dd/MM/yy, it will be changed to Date 1st jan 18 instead of throwing parse error (should be complaining that 13 is not a valid month)